data/categories.txt

```diff
@@ -5378,7 +5378,7 @@ nl:3^Auto reparatie|auto|wagen
 fi:^Autokorjaamo
 fr:3^Réparation d'automobiles|3garage|4station service|voiture|auto
 de:3^Autowerkstatt|Kfz|Reparaturwerkstatt|Auto|Werkstatt
-hu:3^Gépkocsi szervíz|4javítóműhely|kocsi
+hu:3^Autószerelő|Autószerviz|Gépkocsiszerviz|Autójavító|4javítóműhely|kocsi
```

Comment:
Should we really remove the old synonym?

Reply:
It is there but written correctly: _Gépkocsiszerviz_ (the third expression, again, written without spaces)